src/selectedtests/work_items/work_items_cli.py

```diff
@@ -117,10 +117,11 @@ def create_task_mapping(ctx: Context, project: str, src_regex: str, build_regex:
 @click.pass_context
 def process_task_mappings(ctx: Context, weeks_back: int) -> None:
     """Process task mapping work items that have not yet been processed."""
-    # For task mappings, after_date is compared against evergreen version create_time, which is
-    # stored in a UTC date format that is not UTC offset-aware. So after_date does not need to be
-    # offset-aware.
-    after_date = datetime.utcnow() - timedelta(weeks=weeks_back)
+    # after_date is compared against evergreen version create_time, which is stored in a UTC date
+    # format that is UTC offset-aware. So after_date needs to be offset-aware, which is why we
+    # add tzinfo below.
+    now = datetime.utcnow().replace(tzinfo=pytz.UTC)
+    after_date = now - timedelta(weeks=weeks_back)
```

Comment:
fixes error `File \"/usr/local/lib/python3.7/site-packages/selectedtests/task_mappings/version_limit.py\", line 35, in check_version_before_limit\n    if version.create_time < self.stop_at_date:\nTypeError: can't compare offset-naive and offset-aware datetimes"`